jest-e2e.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   preset: 'react-native',
   transform: {
     ...tsjPreset.transform,
-    '\\.js$': '<rootDir>/node_modules/react-native/jest/preprocessor.js',
+    '\\.js$': 'ts-jest',
```

Comment:
not sure why we needed this but it's generating this warning (see the logs on circleci):

```
ts-jest[jest-transformer] (WARN) Got a `.js` file to compile while `allowJs` option is not set to `true` (file: /home/circleci/italia-app/node_modules/react-native/jest/setup.js). To fix this:
  - if you want TypeScript to process JS files, set `allowJs` to `true` in your TypeScript config (usually tsconfig.json)
  - if you do not want TypeScript to process your `.js` files, in your Jest config change the `transform` key which value is `ts-jest` so that it does not match `.js` files anymore
```

@fpersico it's probably safe to remove?